src/examples/chrome/SheetPlacement.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useCallback, useState } from 'react';
+import styled from 'styled-components';
+import { Sheet } from '@zendeskgarden/react-chrome';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Field, Toggle, Label } from '@zendeskgarden/react-forms';
+import { getColor, mediaQuery } from '@zendeskgarden/react-theming';
+
+const StyledField = styled(Field)`
+  margin: ${props => props.theme.space.md};
+`;
+
+const StyledRow = styled(Row)`
+  border: ${props => props.theme.borderWidths.sm} dashed;
+  border-color: ${props => getColor('neutralHue', 400, props.theme)};
+`;
+
+const StyledSheet = styled(Sheet)`
+  ${props => mediaQuery('down', 'sm', props.theme)} {
+    width: 100%;
+
+    /* sheet inner wrapper */
+    & > div {
+      min-width: 100%;
+    }
+  }
+`;
+
+const Example = () => {
+  const [placement, setSheetPlacement] = useState<'start' | 'end' | undefined>('start');
+
+  const toggleSheetPlacement = useCallback(() => {
+    if (placement === 'end') setSheetPlacement('start');
+    if (placement === 'start') setSheetPlacement('end');
+  }, [placement]);
+
+  return (
+    <Grid gutters={false}>
+      <Row>
+        <Col>
+          <StyledField>
+            <Toggle checked={placement === 'start'} onChange={toggleSheetPlacement}>
+              <Label>Place at start of container</Label>
+            </Toggle>
+          </StyledField>
+        </Col>
+      </Row>
+
+      <StyledRow justifyContent={placement}>
+        <StyledSheet isOpen placement={placement}>
+          <Sheet.Header>
+            <Sheet.Title>Placement</Sheet.Title>
+            <Sheet.Description>From side to side</Sheet.Description>
```

Comment:
Needed to deviate from the original content here due to axe testing and the need for unique `aria-labelledby` & `aria-describedby` values. Will need a content check here, and happy to update to what we want.

Reply:
To be fair, the axe warnings here are a trailing concern. The leading concern is aria attribute management on the component itself. I would advocate for allowing the warnings to show here (and honoring the content design), as they are representative of an underlying component issue.